<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="title">
     <string>AVISO</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Ferramenta ainda em fase de desenvolvimento... Após o renomeamento dos arquivos, não será possível desfazer o processo. Por isso use com cuidado.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Diretório</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_diretorio"/>
     </item>
     <item>
      <widget class="QPushButton" name="push_selecionar">
       <property name="text">
        <string>Selecionar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>671</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Arquivos listados</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_arquivos"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Pseudo-nome:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_pseudo_nome"/>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_unico">
         <property name="text">
          <string>Único arquivo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_todos">
         <property name="text">
          <string>Todos</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_executar">
         <property name="text">
          <string>Executar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>671</width>
      <height>217</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Arquivos renomeados</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_renomeados"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
